src/Resources/app/storefront/src/paypal-v2/payone-payment.paypal-v2.js

```diff
@@ -0,0 +1,98 @@
+import Plugin from 'src/plugin-system/plugin.class';
+import HttpClient from "src/service/http-client.service";
+
+export default class PayonePaymentPayPalV2 extends Plugin {
+    static options = {
+        sandbox: true,
+        clientId: null,
+        merchantId: null,
+        currency: null,
+        locale: null,
+        showPayLaterButton: false,
+        createCheckoutSessionUrl: null,
+        onApproveRedirectUrl: null,
+        onCancelRedirectUrl: null,
+        onErrorRedirectUrl: null,
+    };
+
+    init() {
+        if (!this.options.clientId || !this.options.merchantId || !this.options.currency || !this.options.locale || !this.options.createCheckoutSessionUrl || !this.options.onApproveRedirectUrl || !this.options.onCancelRedirectUrl || !this.options.onErrorRedirectUrl) {
+            console.error('The PayPal v2 Express button could not be initialized because the required options are missing.');
+
+            return;
+        }
+
+        this._client = new HttpClient();
+
+        this._loadScript();
+    }
+
+    _loadScript() {
+        const queryString = `client-id=${this.options.clientId}&merchant-id=${this.options.merchantId}&currency=${this.options.currency}&intent=authorize&locale=${this.options.locale}&commit=true&vault=false&disable-funding=card,sepa,bancontact${this.options.showPayLaterButton ? '&enable-funding=paylater' : ''}`;
+        const scriptTag = document.createElement('script');
+
+        scriptTag.onload = () => this._renderButtons();
+
+        if (this.options.sandbox) {
+            scriptTag.src = `https://sandbox.paypal.com/sdk/js?${queryString}`;
+        } else {
+            scriptTag.src = `https://www.paypal.com/sdk/js?${queryString}`;
+        }
+
+        document.head.append(scriptTag);
+    }
+
+    _renderButtons() {
+        paypal.Buttons({
+
```

Comment:
could you remove the many empty rows below, please? (till line 63)

Reply:
Done